static/src/javascripts-legacy/projects/commercial/modules/creatives/add-viewability-tracker.js

```diff
@@ -2,6 +2,13 @@ define(function () {
     return addViewabilityTracker;
 
     function addViewabilityTracker(adSlot, creativeId, viewabilityTracker) {
-        adSlot.insertAdjacentHTML('beforeend', viewabilityTracker.replace('INSERT_UNIQUE_ID', creativeId));
+
+      /*
+        we need to ensure that any scripts in the viewabilityTracker are parsed
+        and executed; this can be done with a Range and ContextualFragment
+      */
+      var range = document.createRange();
+      range.setStart(adSlot, 0);
```

Comment:
I worked on Scribe before joining the Guardian, and they do a lot of `Range` arithmetic. Behaviour is [_extremely_ inconsistent](https://github.com/guardian/scribe/blob/master/BROWSERINCONSISTENCIES.md) across browsers, so assume you have to do everything manually in general.